en/CONTRIBUTING.md

```diff
@@ -1,3 +1,50 @@
 # Contribution Guidelines
 
-Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+:sparkles::heart: Thank you so much for taking the time to contribute! :heart::sparkles:
+
+The following is a set of guidelines for contributing to any of our projects, which are hosted in the [FolksWhoCode Organization](https://github.com/folkswhocode) on GitHub. Feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct and Values](#code-of-conduct-and-values)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+
+## Code of Conduct and Values
+
+Please note that this project is released with a [Contributor Code of Conduct](../CODE_OF_CONDUCT.md)
+and organization's [values](../VALUES.md). By participating in this project you agree to abide by its terms.
+
+## How Can I Contribute?
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue :smile:
+
+### Your First Contribution
+
+You can start by looking through these `beginner-friendly`, `good-first-issue` and `help-wanted` issues:
+
+* [Beginner issues][beginner-friendly]
```

Comment:
Maybe it's good to complete these example labels with some information about what they mean, right? 😄

Reply:
Sure. I'll get right on it 💪